<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgCessCtrl</class>
 <widget class="QDialog" name="dlgCessCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="mnemo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>240</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>135</width>
          <height>135</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame
{
	border-image: url(:/mnemo/CessPool.svg);
	background-repeat: no-repeat;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QProgressBar" name="s_pV">
         <property name="geometry">
          <rect>
           <x>223</x>
           <y>38</y>
           <width>21</width>
           <height>96</height>
          </rect>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QCheckBox" name="s_I_11">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>11</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/butons/pict/bool_green_true_48x48.png);
}

QCheckBox:unchecked
{
	image: url(:/butons/pict/bool_green_false_48x48.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QSlider" name="s_SP_max">
         <property name="geometry">
          <rect>
           <x>241</x>
           <y>34</y>
           <width>20</width>
           <height>104</height>
          </rect>
         </property>
         <property name="whatsThis">
          <string>Постійне завдання на перший параметер</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:vertical {
     background: rgba(0, 0, 0, 0);
/*     position: absolute;  абсолютная позиция в 4px слева и справа от виджета. установка полей виджета также будет работать... */
/*     left: 4px; right: 4px;*/
 }

 QSlider::handle:vertical {
	
	image: url(:/slider/pict/SP_1.png);
 }
</string>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>40</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
        <widget class="QSlider" name="s_SP_min">
         <property name="geometry">
          <rect>
           <x>206</x>
           <y>34</y>
           <width>20</width>
           <height>104</height>
          </rect>
         </property>
         <property name="whatsThis">
          <string>Розрахункове завдання</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:vertical {
     background: rgba(0, 0, 0, 0);
/*     position: absolute;  абсолютная позиция в 4px слева и справа от виджета. установка полей виджета также будет работать... */
/*     left: 4px; right: 4px;*/
 }

 QSlider::handle:vertical {
	image: url(:/slider/pict/SPR_1.png);
 }
</string>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>80</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="s_lV">
         <property name="geometry">
          <rect>
           <x>211</x>
           <y>12</y>
           <width>45</width>
           <height>24</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="s_I_09">
         <property name="geometry">
          <rect>
           <x>283</x>
           <y>142</y>
           <width>25</width>
           <height>46</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 25px;
min-height: 46px;
max-width: 25px;
max-height: 46px;
}
QCheckBox:checked
{
	image: url(:/butons/pict/klapan_diskrette_vert_on_na_25x26.png);
}

QCheckBox:unchecked
{
	image: url(:/butons/pict/klapan_diskrette_vert_off_na_25x46.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="s_V_26_t">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>132</y>
           <width>113</width>
           <height>24</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>67</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Завдвння на рівень</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Мінімум</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="s_sSP_min">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Максимум</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="s_sSP_max">
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Час відстою</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="s_V_26_tz">
        <property name="displayFormat">
         <string>hh:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Управління</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Режим</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="s_Am_Q_09">
        <item>
         <property name="text">
          <string>Руч</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Авт</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Клапан</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QComboBox" name="s_Q_09">
        <item>
         <property name="text">
          <string>Закрито</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Відкрито</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="Close">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Close</sender>
   <signal>clicked()</signal>
   <receiver>dlgCessCtrl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
